src/pages/Profile.js

```diff
@@ -0,0 +1,39 @@
+import Table from 'react-bootstrap/Table';
+import { useSelector } from 'react-redux';
+import '../stylesheets/Profile.css';
+import ListItem from '../components/ProfileList';
+
+const Profile = () => {
+  const missions = useSelector((state) => state.mission);
+  const resultMissions = missions.filter((mission) => mission.reserved);
+  const rockets = useSelector((state) => state.rocket);
+  const resultRockets = rockets.filter((rocket) => rocket.reserved);
+  return (
+    <div className="profile">
+      <Table bordered>
+        <thead>
+          <tr>
+            <th>My Missions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {resultMissions.map((mission) => (
+            <ListItem key={mission.id} id={mission.id} name={mission.name} />))}
+        </tbody>
```

Comment:
[REQUIRED] Kindly use `filter` instead of `map`

Reply:
@fobadara We intended to use filter if you see in line 8 and 10:
`  const missions = useSelector((state) => state.mission);
  const resultMissions = missions.filter((mission) => mission.reserved);
  const rockets = useSelector((state) => state.rocket);
  const resultRockets = rockets.filter((rocket) => rocket.reserved);`
And then use map to turn the data into JSX. Could you please explain further what you mean by use `filter` instead of `map`? I don't understand how they could replace one another. I think I need both.

Thank you in advance.